### PR TITLE
Demo: misc cleanup

### PR DESCRIPTION
**Pull request by danrabbit**
* GObject style
* Remove unnecessary properties
* Clarify "Replaces ID"
* Organize
* Use `attach` for fewer grids
* Correct label case
* Don't use combobox text for logic (that's dangerous if it ever changes or is translated)
* Suggested action button and align

### CODE REVIEW

File: demo/MainWindow.vala [hunk 1]
```diff
@@ -27,85 +27,83 @@ public class MainWindow : Gtk.ApplicationWindow {
...
-        case "high":
+        case 2:
             priority = NotificationPriority.HIGH;
             break;
-        case "low":
+        case 1:
             priority = NotificationPriority.LOW;
             break;
-        case "normal":
+        case 0:
```
Comment (meisenzahl) [inline demo/MainWindow.vala:99]:
```suggestion
        case 1:
```

File: demo/MainWindow.vala [hunk 2]
```diff
@@ -27,85 +27,83 @@ public class MainWindow : Gtk.ApplicationWindow {
...
+        switch (priority_combobox.active) {
+        case 3:
             priority = NotificationPriority.URGENT;
             break;
-        case "high":
+        case 2:
             priority = NotificationPriority.HIGH;
             break;
-        case "low":
+        case 1:
```
Comment (meisenzahl) [inline demo/MainWindow.vala:96]:
```suggestion
        case 0:
```

**Review by meisenzahl — COMMENTED** [review]
Wouldn't the two have to be swapped?

**Comment by danrabbit** [comment]
@meisenzahl you're right, I can't read. Thanks :smile:

**Review by meisenzahl — APPROVED** [review]

**Pull request merged**